src/pages/nft/class/_classId/index.vue

```diff
@@ -151,7 +151,11 @@ export default {
             this.NFTImageUrl || 'https://liker.land/images/og/writing-nft.jpg',
         },
       ],
-      link: [{ rel: 'canonical', href: `${this.$route.path}` }],
+      link: [
+        { rel: 'canonical', href: `${this.$route.path}` },
+        { rel: 'preload', href: 'https://js.stripe.com/v3', as: 'script' },
+      ],
+      script: [{ src: 'https://js.stripe.com/v3', async: true }],
```

Comment:
we better inject this script whenever a fiat payment can be triggered, but I'm not sure how